core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CloudflowSparkPlugin.scala

```diff
@@ -25,20 +25,22 @@ import spray.json._
 
 import cloudflow.blueprint.StreamletDescriptorFormat._
 import cloudflow.sbt.CloudflowKeys._
+import CloudflowBasePlugin._
 
-object CloudflowSparkPlugin extends CloudflowBasePlugin {
+object CloudflowSparkPlugin extends AutoPlugin {
   final val sparkVersion                  = "2.4.4"
   final val cloudflowVersion              = "1.3.1-SNAPSHOT"
```

Comment:
@debasishg  How are we going to replace this hardcoded version number during a release?

Reply:
Yeah .. we can have it in the base plugin but that does not solve the problem though. But a bigger question which we started discussing is if we need to have the cloudflow version as part of the the base image name.

Reply:
Yes, I still had a tab open with the following comment:

> Here's my €0.02: 
> 
> I think the issue here is mostly related to release lifecycles and the question whether that is different for base images vs the other project artifacts.
> 
> I do think there is value in marking the base images that are part of an official release with the verified version number of that release, even though that might mean recreating the same image but with a different tag for releases where nothing has changed in the base image.
> 
> That said, I think that should be a special case and done as part of our release automation, which can also update existing example projects to use the